packages/frontend/src/pages/positions/PositionDetails.tsx

```diff
@@ -3,24 +3,42 @@ import { formatUSDCents } from '../formatUSDCents'
 import { Page } from '../common/Page'
 import { Table } from '../common/Table'
 import { PositionDetailsProps } from './PositionDetailsProps'
+import { assetIcons } from '../common/icons/assets/assetIcons'
 
-const assetTableColumns = [
-  { header: 'Asset id' },
+type AssetCellProps = {
+  assetId: string
+}
+
+function AssetCell({ assetId }: AssetCellProps) {
+  const name = assetId.replace(/-\d+$/, '')
+  const Icon = assetIcons[name]
+  if (!Icon) {
+    return <>{name}</>
+  }
```

Comment:
I dislike the `.replace` call. The `assetIcons` map should also be a local thing and the code for the icon should look like this:
```ts
<AssetIcon assetId={assetId} />
```
When it comes to the name, i think we should have a function called `AssetId.name(assetId)`. and it can be `return assetId.split('-')[0]`